app/views/course/assessment/submissions/_submission_experience_points_reason.html.slim

```diff
@@ -0,0 +1,2 @@
+- submission = submission_experience_points_reason
+= format_inline_text(submission.assessment.title)
```

Comment:
Do we want to link this back to the submission as well? I think that is required.